tests/test_individually.py

```diff
@@ -90,3 +90,14 @@ def test_neal(self):
         J = {('a', 'b'): -1}
 
         resp = neal.SimulatedAnnealingSampler().sample_ising(h, J)
+
+    def test_preprocessing(self):
+        import dimod
+        from dwave.preprocessing.lower_bounds import roof_duality
+        from dwave.preprocessing.composites import ScaleComposite
+
+        bqm = dimod.BinaryQuadraticModel.from_ising({'a': 10}, {'ab': -1, 'bc': 1})
+        result = roof_duality(bqm)
```

Comment:
How about a sanity check as well?
```suggestion
        result = roof_duality(bqm)
        self.assertFalse(bqm.variables - result)
```